src/phantom.js

```diff
@@ -219,12 +224,32 @@ export default class Phantom {
      */
     exit() {
         clearInterval(this.heartBeatId);
-        this.execute('phantom', 'invokeMethod', ['exit']);
+        return this.execute('phantom', 'invokeMethod', ['exit']);
+    }
+
+    /**
+     * Clean up and force kill this process
+     */
+    kill(errmsg = 'Phantom process was killed') {
+        this.rejectCommands(errmsg);
+        this.process.kill('SIGKILL');
+        return Promise.reject(errmsg);
     }
 
     _heartBeat() {
         if (this.commands.size === 0) {
             this.execute('phantom', 'noop');
         }
     }
+
+    /**
+     * rejects all commands in this.commands
+     */
+    rejectCommands(errmsg = 'Phantom exited prematurely') {
```

Comment:
Can you make all the methods that shouldn't be used externally to start with `_` so like `_rejectCommands`